tests/integration-tests/clusters_factory.py

```diff
@@ -162,6 +162,10 @@ def create_cluster(self, cluster):
             error = "Cluster creation failed for {0} with output: {1}".format(name, result.stdout)
             logging.error(error)
             raise Exception(error)
+        elif "WARNING" in result.stdout:
+            error = "Cluster creation generated a warning: {1}".format(name, result.stdout)
```

Comment:
in the format you only have the {1} but you're passing two variables

Reply:
fixed

Reply:
I'd add the name of the cluster like in the error case.
`"Cluster creation for {0} generated a warning: {1}".format(name, result.stdout)`

Reply:
done